src/core/annotation.js

```diff
@@ -752,6 +754,26 @@ var TextWidgetAnnotation = (function TextWidgetAnnotationClosure() {
   return TextWidgetAnnotation;
 })();
 
+var ButtonWidgetAnnotation = (function ButtonWidgetAnnotationClosure() {
+  function ButtonWidgetAnnotation(params) {
+    WidgetAnnotation.call(this, params);
+
+    this.data.noToggleToOff = this.hasFieldFlag(
+      AnnotationFieldFlag.NOTOGGLETOOFF);
+    this.data.radio = this.hasFieldFlag(AnnotationFieldFlag.RADIO);
```

Comment:
Do we really want to check that the document complies with the PDF spec? What does it bring?